indigo/bulk_creator.py

```diff
@@ -51,6 +52,30 @@ def clean_title(self):
         return re.sub('[\u2028 ]+', ' ', title)
 
 
+class RowValFormCap(RowValidationFormBase):
+    """ Includes (optional) Chapter (cap) field.
+    For this field to be recorded on bulk creation, add `'cap': 'Chapter (Cap.)'`
+    for the relevant country in settings.INDIGO['WORK_PROPERTIES']
+    """
+    cap = forms.CharField(required=False)
+
+
+class RowValFormPubDateOpt(RowValidationFormBase):
```

Comment:
Style: avoid cryptic abbreviations. `PublicationDateOptional` or `PublicationDateOptionalRowValidationForm` (verbose, but clear). The former is probably sufficient given that it's in the context of the bulk creator. In fact, we could make this a mixin.

Take a look at how Django does its class naming and hierarchy, they use mixins a bunch: https://ccbv.co.uk/